extensions/amp-analytics/0.1/vendors.js

```diff
@@ -697,6 +697,9 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
           'clt=${contentLoadTime}&' +
           'dit=${domInteractiveTime}' +
           '${baseSuffix}',
+      'user-error': '${host}/r/collect?${basePrefix}&' +
+          'em=${errorMessage}' +
```

Comment:
it does not seem to comply Google Analytics' spec: https://developers.google.com/analytics/devguides/collection/analyticsjs/exceptions

did you test with a real tracking code?